client/src/components/PitchDoc/SubmitPitchModal.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement } from 'react';
+import { Modal, Button, Form, Image } from 'semantic-ui-react';
+
+// import { currentTeamsButtons } from '../../utils/constants';
+import '../../css/pitchDoc/SubmitPitchModal.css';
+import GoogleDriveScreenshot from '../../assets/GoogleDriveScreenshot.png';
+// import WizardSelectButton from '../WizardSelectButton/WizardSelectButton';
+
+// const defaultOnClick = (): void => void 0;
+
+function SubmitPitchModal(): ReactElement {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Modal
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={
+        <Button className="submit-pitch-button"> Submit a Pitch </Button>
+      }
+      closeIcon
+    >
+      <Modal.Actions>
+        <Form>
+          <Form.Field>
+            Pitch Title
+            <label>
+              <input />
+            </label>
+          </Form.Field>
+          <Form.Field>
+            Summarize Pitch in 1-2 Sentences (300 character limit)
+            <label>
+              <input />
+            </label>
+          </Form.Field>
+
+          <div className="suggested-text">Suggested Roles Needed:</div>
+
+          <div className="gdrive-teams-container">
+            <Image src={GoogleDriveScreenshot}></Image>
+
+            <div className="teams-section">
+              <Form.Group widths="equal">
+                {/* {Object.keys(currentTeamsButtons).map((team: string) => (
+                  <div key={team}>
+                    <WizardSelectButton
+                      onClick={defaultOnClick}
+                      key={team}
+                      selectedArray={[]}
+                      width="40px"
+                      margin="10px 15px 10px 15px"
+                      value={team}
+                      color={currentTeamsButtons[team]}
+                      disabled
+                    />
+
+                    <Form.Field>
+                      <input />
+                    </Form.Field>
+                  </div>
+                ))} */}
+              </Form.Group>
+            </div>
+
+            <div className="break"></div>
```

Comment:
use the break tag `<br />` 

Although I'm not sure if this break class has extra functionality, if it does, the css should probably be applied to the class above or below it and not in this empty div.

Also, just say `<div className="break" />`